ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly.yaml

```diff
@@ -2865,6 +2865,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-shared-to-local-gateway-mode-migration
+- as: vsphere-ipi-ovn-winc-p2-f14
+  cron: 32 12 8,23 * *
+  remote_api: true
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@amd64 and @vsphere-ipi and @network-ovnkubernetes and not @fips'
```

Comment:
Comparing with the[ WINC existing non-nightly jobs](https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14.yaml#L208C1-L220C35) I guess this job won't trigger any Ginkgo test, right? It's running only cucushift tests. Is that correct?

Reply:
Ok, I can see all the [ginkgo tests being triggered](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/40561/rehearse-40561-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-ovn-winc-p2-f14/1676138603555590144/artifacts/vsphere-ipi-ovn-winc-p2-f14/openshift-extended-test/build-log.txt). However, I don't see any Windows Containers tests being executed. Is that expected? I guess the TEST_SCENARIOS needs some tuning in.

Reply:
yes some tags might be needed for targetting specific tests, not sure at the moment.

Reply:
I will take care of adding them in a follow up patch, but I believe this is a mistake we had in our Jenkins profiles. Windows_Containers tests [were not run in vSphere WINC profiles](https://gitlab.cee.redhat.com/aosqe/ci-profiles/-/blob/master/4.14/73_IPI%20on%20vSphere%207.0%20%26%20OVN%20%26%20WindowsContainer%20%26%20thin%20pvc.test_run.yaml#L20), however they were included in all the other WINC profiles (AWS, GCP and Azure). To avoid this occurs again we need to make sure the Windows_Containers scenario gets added to this job too, as currently [no Windows Containers test gets triggered](https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/pr-logs/pull/openshift_release/40561/rehearse-40561-periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-vsphere-ipi-ovn-winc-p2-f14/1678399019644096512/artifacts/vsphere-ipi-ovn-winc-p2-f14/openshift-extended-test/build-log.txt):
`final scenarios: MCO|ETCD|PSAP|SDN|STORAGE|API_Server|Authentication|Cluster_Operator|Cluster_Infrastructure|OLM|Network_Edge|Operator_SDK|Workloads|Image_Registry|Container_Engine_Tools|NODE|OTA|PerfScale|Cluster_Observability|Security_and_Compliance|LOGGING`

Reply:
Thanks @jfrancoa , acked.

Reply:
I'm returning to this...as I really don't think it adds any value to have a vSphere Windows profile that does run every other single test, but no Windows containers test at all.
Currently, the only Windows tests are based on Ginkgo (no cucushift tests were created for WMCO) and to re-run the same set of tests that other profiles run I think it would be wasting resources.
I am working currently in enabling Windows profiles also for AWS and GCP and this is the planned configuration:
```
      TEST_SCENARIOS: Windows_Containers
      FILTERS_ADDITIONAL: Smokerun&
      TEST_TIMEOUT: "50"
      TAG_VERSION: '@4.14'
test:
    - ref: openshift-extended-test
```
Triggering only the Smokerun subet (around one hour) tests which will provide feedback on the WMCO tests without taking too long. Also, this Smokerun subset is a pretty stable set of tests, so we won't have unstable failures being reported to ReportPortal. Don't you think it would make more sense to run this instead of all the cucushift plus all the ginkgo tests for other areas (leaving Windows_Containers tests aside).